package.json

```diff
@@ -23,11 +23,12 @@
   },
   "scripts": {
     "build": "gulp",
-    "pretest": "npm run lint",
-    "test": "gulp test",
-    "html-report": "nyc report --temp-directory=./coverage --reporter=html",
-    "jest": "jest --coverage",
-    "coveralls": "nyc report --temp-directory=./coverage --reporter=text-lcov | coveralls",
+    "pretestonly": "chromedriver &",
```

Comment:
I prefer to use headless browsers for local tests if possible. I wonder if the current version of phantomjs is stable enough or can use other one.